.github/workflows/sycl-linux-precommit.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           - name: Intel Arc A-Series Graphics
             runner: '["Linux", "arc"]'
             image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
-            target_devices: level_zero:gpu;opencl:gpu;level_zero_v2:gpu
+            target_devices: level_zero:gpu;opencl:gpu
```

Comment:
can you add more info to the PR description of why we are no longer testing on l0v2 in CI?